### PR TITLE
Add error msg for null file while Adding Commercial for event

### LINKED ISSUE #1959: Null file validation required on adding commercials
**I'm submitting a ..**
- [x] Bug Report

**Current behavior:**
Null file validation required while adding commercials
screenshot from 2018-01-12 17-00-36

**Steps to reproduce:**
goto : '/admin/conferences/tester/program/events' > Add Commercials > Add

### PR DESCRIPTION
**Pull request by AnkushMalik**
Null file error message added for uploading empty file in CommercialsController#mass_upload

Closes #1959

**Checklist**

- [x] I have read the [Contribution & Best practices Guide](https://github.com/openSUSE/osem/blob/master/CONTRIBUTING.md).
- [x] My branch is up-to-date with the upstream `master` branch.
- [x] The tests pass locally with my changes.
- [ ] I have added tests that prove my fix is effective or that my feature works(if appropriate).
- [ ] I have added necessary documentation (if appropriate).

**Short description of what this resolves/which [issues](https://github.com/openSUSE/osem/issues) does this fix?:**

- Null file error msg required on adding commercials, #1959

### CODE REVIEW

File: app/controllers/admin/commercials_controller.rb [hunk 1]
```diff
@@ -57,7 +57,9 @@ def render_commercial
     def mass_upload
       errors = Commercial.read_file(params[:file]) if params[:file]
 
-      if errors.all? { |_k, v| v.blank? }
+      if !errors
```
Comment (bear454) [inline app/controllers/admin/commercials_controller.rb]: Are there any other conditions that could cause an error here?

Comment (AnkushMalik) [reply app/controllers/admin/commercials_controller.rb]: yes when user clicks on add commercial without selecting any file 'errors' in line 58^ is nil, it redirects him to error page: 



So, i thought there should be a error msg and redirect him back

Comment (AnkushMalik) [reply app/controllers/admin/commercials_controller.rb]: @bear454, is there any problem with this issue or PR? can you please review this :-) ?

Comment (bear454) [reply app/controllers/admin/commercials_controller.rb]: I think you misunderstood my question - I read your code as treating _any error_ as a result of a bad file upload; have you considered that _other_ conditions may cause errors ?

Comment (AnkushMalik) [reply app/controllers/admin/commercials_controller.rb]: oh, my bad, now I understand my mistake thanks :-)

**Pull request merged**